client/src/main/java/io/pravega/client/admin/StreamManager.java

```diff
@@ -238,10 +238,10 @@ static StreamManager create(ClientConfig clientConfig) {
      *
      * @param scopeName The scope of the stream.
      * @param streamName The stream name.
-     * @return stream information.
+     * @return A future representing {@link StreamInfo}.
```

Comment:
Java documentation can be updated with more details on exception thrown incase of stream/scope which does not exist in the system.